filebeat.yml

```diff
@@ -13,6 +13,7 @@ filebeat.prospectors:
   document_type: kube-logs
   fields:
     host: ${FILEBEAT_HOST:${HOSTNAME}}
+    k8s-cluster: ${K8S_CLUSTER}
```

Comment:
We should probably set a default value for this? Something like `default` maybe?

Reply:
Also, I think it should be `k8s_cluster` or `cluster_name` since Elasticsearch splits the terms at `-`.

Reply:
You are absolutely right, there you go :)